DEVELOPMENT.md

```diff
@@ -24,6 +24,23 @@ Run the following to invoke those tests:
 pytest --local-image-id REPLACE_ME_WITH_IMAGE_ID
 ```
 
+## Code Style
+
+Install pre-commit to run code style checks before each commit:
+
+```shell
+pip install pre-commit
```

Comment:
given we have instructed to conda install the environment.yml which includes pre-commit, we don't need to additionally pip install the package.